docs/Setup.md

```diff
@@ -40,18 +41,23 @@ pip install azureml-core azureml-mlflow
 ``` powershell
 pip install -e src/azure-ml-rai
 ```
-6. Create a JSON file called component_config.json containing: '{version:1}'. This will track the version of the components that you are registering in your workspace and will be needed when you run a jupyter notebook.
+10. Create a JSON file called component_config.json containing: '{version:1}'. This will track the version of the components that you are registering in your workspace and will be needed when you run a jupyter notebook.
 ```powershell
 echo {version:1} > component_config.json
 ```
 
-8. Run the following command to register the private preview components in your workspace
+11. Run the following command to register the private preview components in your workspace
 
 ```powershell
-scripts/Register-AzureML.ps1 src/responsibleai/registration_config.json
-
+scripts/Register-AzureML.ps1 src/responsibleai
+```
+```powershell
+scripts/Register-AzureML.ps1 test
 ```
-9. Validate that your components have been registered in your workspace at https://ml.azure.com
+
+12. Validate that your components have been registered in your workspace at https://ml.azure.com
```

Comment:
Do this by.... going to the Components tab, and looking for entries like "Gather RAI Insights Dashboard"

Reply:
updated